fnl/paperplanes/providers/init.fnl

```diff
@@ -2,4 +2,5 @@
  :ix.io (require :paperplanes.providers.ixio)
  :0x0.st (require :paperplanes.providers.0x0st)
  :sprunge.us (require :paperplanes.providers.sprungeus)
- :paste.rs (require :paperplanes.providers.pasters)}
+ :paste.rs (require :paperplanes.providers.pasters)
+ :srht (require :paperplanes.providers.srht)}
```

Comment:
Other mirror the url, so probably "sr.ht" or "pastes.sr.ht" (probably a bit much).

Reply:
done